implementations/contracts/ERC725/IERC725X.sol

```diff
@@ -20,8 +20,14 @@ interface IERC725X  /* is ERC165, ERC173 */ {
     /**
     * @dev Emitted when a contract executed.
     */
-    event Executed(uint256 indexed _operation, address indexed _to, uint256 indexed  _value, bytes _data);
+    event Executed(Operation indexed _operation, address indexed _to, uint256 indexed  _value, bytes _data);
```

Comment:
The Interface i would actually like to keep as is, as the ENUM is a simple uint256, and this might change the function selector.

Also the Operation ENUMS will get more in the future, and that shouldn't change the standard interface.